config/file.v1/file.go

```diff
@@ -131,6 +133,14 @@ func (file File) JIRAProjectKey() string {
 	return file.CLI.JIRAProjectKey
 }
 
+func (file File) Link() string {
+	return file.CLI.Link
```

Comment:
Note that these properties will not automatically be written to the configuration file unless you explicitly update `config.WriteFile`.

(In either case, we shouldn't be writing the link to the file anyway, since we expect it to change for every build.)